.flake8-packaged-files

```diff
@@ -10,6 +10,12 @@ ignore =
     D202, # No blank lines allowed after function docstring
 ### Temporarily disabled docstring errors:
     D105, # missing docstring in magic method
-    I401, # Missing exception(s) in Raises section
-    D402, # First line should not be the function’s “signature”
 show-source = True
+
+### Ignore docstring complaints in data models
+per-file-ignores =
+    settings.py:D101,D102,D106,D107
+    settings_2d.py:D101,D102,D106,D107
+    camera_state.py:D101,D102,D106,D107
+    camera_info.py:D101,D102,D106,D107
+    frame_info.py:D101,D102,D106,D107
```

Comment:
You probably want to add a comment about what kind of errors D101, D102, D106 and D107 really are. Makes it easier to fix them in the right order later

Reply:
Good idea, I'll do that.

Reply:
Done

Reply:
Nice